### ISSUE TITLE
Nightly broken

### ISSUE THREAD

**Issue by nikku** [camunda/camunda-modeler#1128]
Nightly of the app is currently not being built (cf. [CI build](https://travis-ci.org/camunda/camunda-modeler/builds/482627013)):

```
lerna ERR! ENOGIT Detached git HEAD, please checkout a branch to choose versions.
/project/node_modules/execa/index.js:347
		throw err;
		^
Error: Command failed: lerna publish --repo-version=3.0.0-nightly --skip-npm --skip-git --yes
    at makeError (/project/node_modules/execa/index.js:174:9)
    at module.exports.sync (/project/node_modules/execa/index.js:338:15)
    at Object.<anonymous> (/project/tasks/distro.js:41:3)
    at Module._compile (internal/modules/cjs/loader.js:721:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:732:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:774:12)
    at executeUserCode (internal/bootstrap/node.js:342:17)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! camunda-modeler-builder@0.0.0 distro: `node tasks/distro.js "--win" "--linux" "--nightly" "--publish"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the camunda-modeler-builder@0.0.0 distro script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

__Tasks__

* [ ] Investigate solution that works with `lerna@3` (https://github.com/camunda/camunda-modeler/issues/1128#issuecomment-456400672)
* [ ] Fix Camunda Modeler nightly build (https://github.com/camunda/camunda-modeler/pull/1129)
* [ ] Fix Zeebe Modeler nightly build (broken, too) (https://github.com/zeebe-io/zeebe-modeler/pull/46)

**Comment by nikku** [comment]
After looking through literally 100 different issues in the `lerna` issue tracker I can confirm that our use-case is too esoteric to be supported out of the box. It was well supported by `lerna@2` is not any more in `lerna@3`

__What happens__

* Travis does not check out a branch but a `detached HEAD` commit when it executes a CRON job
* `lerna version` does some magic (don't know which exactly) and complains about the detached head

__How we fix things__

As a fix / workaround we simply [checkout the detached commit onto it's own temporary branch](https://github.com/camunda/camunda-modeler/pull/1129/commits/f9832f35c72fdee64c45ac084edb9e2a011e0378) :carousel_horse:.

**Comment by pinussilvestrus** [comment]
Fixed via #1129 and zeebe-io/zeebe-modeler#46. Let's see on the next regular nightly whether it actually worked live.
